.github/workflows/build.yaml

```diff
@@ -507,9 +538,10 @@ jobs:
 
     - name: Push to quay.io/rhacs-eng
       uses: docker/build-push-action@v4
+      if: |
+        github.event_name == 'push' || !github.event.pull_request.head.repo.fork
```

Comment:
Is there any way to extract it to a variable?

Reply:
Didn't find a way to extract these things to variables (especially since they somewhat rely on the context of the github object) 😞